fluent-react/test/localized_overlay.test.js

```diff
@@ -850,8 +855,7 @@ foo = test <em>custom markup parser</em>
     function parseMarkup(str) {
       return [
         {
-          TEXT_NODE: 3,
-          nodeType: 3,
+          nodeName: "#text",
```

Comment:
I simplified the detection of text nodes relying on the fact that their `nodeName` is guaranteed by the standard to be `#text`.